### ISSUE TITLE
Incompatible with (unofficial) Rust discord-sdk

### ISSUE THREAD

**Issue by leo60228** [OpenAsar/arrpc#67]
The Rust discord-sdk crate [assumes all messages have a `nonce` field](https://github.com/EmbarkStudios/discord-sdk/blob/7e0eca4fdefa642aecc00f8e943e26cd0d2ab6ab/sdk/src/handler.rs#L184). However, arrpc's [initial `READY` event](https://github.com/OpenAsar/arrpc/blob/3e22fd776273afaa4a80c51deb86077ffdd4d2ae/src/server.js#L32-L54) does not include this field, resulting in this error:

```
2024-01-15 22:18:20,122 DEBUG [discord_sdk::io] connected to /run/user/1000/discord-ipc-0!
2024-01-15 22:18:20,133 WARN [discord_sdk::handler] Failed to deserialize message: missing field `nonce` at line 1 column 301 {"cmd":"DISPATCH","evt":"READY","data":{"v":1,"user":{"id":"1045800378228281345","username":"arRPC","discriminator":"0000","avatar":"cfefa4d9839fb4bdf030f91c2a13e95c","flags":0,"premium_type":0},"config":{"api_endpoint":"//discord.com/api","cdn_host":"cdn.discordapp.com","environment":"production"}}}
2024-01-15 22:18:20,133 ERROR [mpdiscord::discord] discord error: missing field `nonce` at line 1 column 301
```

**Comment by CanadaHonk** [comment]
Sorry just saw, thank you for this great catch and details! Fixed in 981e6bb5d72437b196e6597b21a71e3fe8e687ce

**Comment by leo60228** [comment]
It seems like discord-sdk expects specifically a string that can be parsed as a `usize`; should the `"initial-nonce"` perhaps be changed to `"0"` or similar?
